src/docs/sphinx/quickstart_guide/config_build.rst

```diff
@@ -276,76 +307,152 @@ directly to CMake; for example::
 | AXOM_ENABLE_ALL_COMPONENTS   | ON      | Enable all components by default       |
 +------------------------------+---------+----------------------------------------+
-| AXOM_ENABLE_<FOO>            | ON      | Enables the axom component named 'foo' |
+| AXOM_ENABLE_<FOO>            | ON      | Enable the axom component named 'Foo'  |
 |                              |         |                                        |
 |                              |         | (e.g. AXOM_ENABLE_SIDRE)               |
-|                              |         | for the sidre component                |
+|                              |         | for the Sidre component                |
 +------------------------------+---------+----------------------------------------+
-| AXOM_ENABLE_DOCS             | ON      | Builds documentation                   |
+| AXOM_ENABLE_EXAMPLES         | ON      | Build Axom examples                    |
 +------------------------------+---------+----------------------------------------+
-| AXOM_ENABLE_EXAMPLES         | ON      | Builds examples                        |
+| AXOM_ENABLE_TESTS            | ON      | Build Axom unit tests                  |
 +------------------------------+---------+----------------------------------------+
-| AXOM_ENABLE_TESTS            | ON      | Builds unit tests                      |
+| ENABLE_BENCHMARKS            | OFF     | Enable Google benchmark                |
 +------------------------------+---------+----------------------------------------+
-| AXOM_ENABLE_TOOLS            | ON      | Builds tools                           |
+| AXOM_ENABLE_DOCS             | ON      | Enable Axom documentation to be built  |
+|                              |         | as a make target                       |
 +------------------------------+---------+----------------------------------------+
-| BUILD_SHARED_LIBS            | OFF     | Build shared libraries.                |
-|                              |         | Default is Static libraries            |
+| AXOM_ENABLE_TOOLS            | ON      | Enable Axom development tools          |
++------------------------------+---------+----------------------------------------+
+
+If ``AXOM_ENABLE_ALL_COMPONENTS`` is OFF, you must explicitly enable the desired
+components (other than 'core', which is always enabled).
+
+See `Axom software documentation <../../../index.html>`_
+for a list of Axom components and their dependencies. Note that when enabling 
+an external dependency for an Axom component, the CMake variable ``BAR_DIR`` 
+must be set to a valid path to the dependency installation. See 
+:ref:`dependencies-label` for a complete listing of configuration variables 
+to specify paths to Axom external dependencies. 
+
+.. note:: ``AXOM_ENABLE_EXAMPLES``, ``AXOM_ENABLE_TESTS``, and ``AXOM_ENABLE_DOCS``
+          are *CMake-dependent options*. Thus, if a non-Axom prefix variable form
```

Comment:
I was confused by this sentence.  Is this what you meant?

```suggestion
          are *CMake-dependent options*. Thus, if a variable without the prefix ``AXOM_``
```

Reply:
Yes, that's what I meant. I changed to your suggestion as it is clearer.